### ISSUE TITLE
Plants Vs Zombies: Game of the Year (3590)

### ISSUE THREAD

**Issue by derram** [ValveSoftware/Proton#238]

**Comment by derram** [comment]
@kisak-valve I upgraded from Xubuntu 16.04 to 18.04 and the sound is working now.

The bit about the launch options is still relevant, though.

**Comment by whizse** [comment]
Also covered by issue #340

**Comment by some-username-here1** [comment]
Given that I tested PvZ with the `-changedir` launch command, it immediately crashed upon Shift+Tab'ing to bring up the Steam overlay, when I was in-game

**Comment by some-username-here1** [comment]
Another issue is that it starts dropping the framerate a ton (below 30) whenever your on the plant selection menu, before starting a new level

**Comment by some-username-here1** [comment]
There's also a thin single-color (such as green, brown, etc.) line that sometimes appears on the bottom- and right-side of the screen when in fullscreen, sometimes it's of the previous scene a couple of seconds ago

Though Alt+Tabbing out and back in clears it up (or not having fullscreen enabled prevents it, I'm assuming)

**Comment by davidak** [comment]
It works without issues since Proton 5.0-8.

https://www.protondb.com/app/3590

**Comment by HenryJk** [comment]
A workaround is possible for eliminating plant selection fps drop though it could possibly a DRM violation. When the game is running, the game creates another binary called popcapgame1.exe in `steamapps/compatdata/3590/pfx/drive_c/ProgramData/PopCap Games/PlantsVsZombies` and then executing the majority of cpu time in this binary. Replacing the original PlantsVsZombies.exe binary with this one seems to solve all stutter problem. However, whether cloud save and achievement is still working is untested using this method.

**Comment by kisak-valve** [comment]
Plants vs Zombies: Game of the Year - Hardware Acceleration

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6940.
@RedSoxFan04 posted on 2023-07-20T18:35:41:

The game runs well, but as far as I can tell, it seems like it's just using software rendering instead of being rendered by the GPU. For reference, I have an AMD RX 5600 XT, which is very well-supported on Linux. I would like to be able to use GPU rendering because it would allow my PC to run much more quietly while playing this game.

**Comment by kisak-valve** [comment]
Hello @RedSoxFan04, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by RedSoxFan04** [comment]
https://gist.github.com/RedSoxFan04/5692f7df33e9e2dfebe31d69692a62fe

**Comment by kisak-valve** [comment]
Plants vs. Zombies GOTY Edition: audio speedup and cutting

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9866.
@DAnaselo posted on 2026-06-09T23:47:37:

# Compatibility Report
- Name of the game with compatibility issues: Plants vs. Zombies GOTY Edition
- Steam AppID of the game: 3590

## System Information
- GPU: Nvidia Geforce GTX 1660 Super
- Video driver version: 610.43.02
- Kernel version: 7.0.11.arch1-1
- Proton version: Proton Experimental (as of 2026-06-01)

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
after playing the game from 30 minutes to 1 hour. the audio speeds up (to a fixed speed), and starts randomly cutting. it only applies to the background game audio, not the audio effects like zombies sounds or peashooters
